compiler/src/dotty/tools/dotc/reporting/diagnostic/ErrorMessageID.java

```diff
@@ -52,6 +52,9 @@
     MixedLeftAndRightAssociativeOpsID,
     CantInstantiateAbstractClassOrTraitID,
     AnnotatedPrimaryConstructorRequiresModifierOrThisID,
+    OverloadedOrRecursiveMethodNeedsResultTypeID,
+    RecursiveValueNeedsResultTypeID,
+    CyclicReferenceInvolvingImplicitID,
```

Comment:
@felixmulder recently changed from directly specified int constants to this Java enum.